sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/aggregator/AbstractMinMaxSumCountAggregator.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.aggregator;
+
+import io.opentelemetry.api.common.Labels;
+import io.opentelemetry.sdk.common.InstrumentationLibraryInfo;
+import io.opentelemetry.sdk.metrics.common.InstrumentDescriptor;
+import io.opentelemetry.sdk.metrics.data.DoubleSummaryData;
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import io.opentelemetry.sdk.resources.Resource;
+import java.util.Map;
+
+abstract class AbstractMinMaxSumCountAggregator
+    extends AbstractAggregator<MinMaxSumCountAccumulation> {
```

Comment:
Since we're coercing all the longs into doubles when we accumulate, is worth having separate aggregators for long & double? Could we get away with just using double?

Reply:
Separate PR? Already large this one and very complicated

Reply:
sounds good!